src/Sentry.AspNetCore/SentryMiddleware.cs

```diff
@@ -18,6 +18,10 @@ namespace Sentry.AspNetCore;
 /// </summary>
 internal class SentryMiddleware : IMiddleware
 {
+    internal const string TraceHeaderItemKey = "sentry-trace-header";
```

Comment:
Why do we have a separate header name here to [the one used in SentryTraceHeader](https://github.com/getsentry/sentry-dotnet/blob/dc0e0223aa64555966ff5c9ab194a535dc737b72/src/Sentry/SentryTraceHeader.cs#L8)?